app/models/concerns/kms/compile_templates.rb

```diff
@@ -9,6 +9,10 @@ module CompileTemplates
       after_commit :compile_templates, on: [:create, :update]
     end
 
+    def register_id
+      (self.class.name << "_" << self.id.to_s).parameterize(separator: "_")
```

Comment:
Ok for moving it into module, but would like to keep it as it was (only pass named keyword 'separator')